src/main/java/games/strategy/engine/framework/GameDataFileUtils.java

```diff
@@ -9,21 +9,22 @@
 
 import com.google.common.annotations.VisibleForTesting;
 
-import games.strategy.engine.ClientContext;
+import games.strategy.triplea.settings.ClientSetting;
 
 /**
  * A collection of utilities for working with game data files.
  */
 public final class GameDataFileUtils {
   private static final IOCase DEFAULT_IO_CASE = IOCase.SYSTEM;
 
-  private static final SaveGameFormat DEFAULT_SAVE_GAME_FORMAT =
-      ClientContext.gameEnginePropertyReader().useNewSaveGameFormat()
-          ? SaveGameFormat.NEW
-          : SaveGameFormat.CURRENT;
-
   private GameDataFileUtils() {}
 
+  private static SaveGameFormat getDefaultSaveGameFormat() {
```

Comment:
> Looking at the helper methods, something looks to have drove a good deal of indirection.

"Something" = unit tests and legacy requirements.

> Regardless, there may be some simplifications here

"Simplification" = some of the indirection and all of the old code disappears when we fully switch to the new format. :smile: